executor/runtime/types/types.go

```diff
@@ -514,8 +516,48 @@ func getKernelVersion() string {
 
 }
 
-func (d Details) RenderVersionDetails() string {
-	return fmt.Sprintf("kernel-version=%s,titus-executor=%s", getKernelVersion(), executor.TitusExecutorVersion)
+var tsaVersion = ""
+
+func getTSAVersion(taskID string) string {
+	if tsaVersion != "" {
+		return tsaVersion
+	}
+
+	serviceName := fmt.Sprintf("titus-sidecar-seccomp-agent@%s.service", taskID)
+	out, err := exec.Command("systemctl", "show", "--property", "MainPID",
+				 "--value", serviceName).CombinedOutput()
+	if err != nil {
+		return fmt.Sprintf("couldn't find tsa version: %v", err)
+	}
+
+	outInt, err := strconv.Atoi(strings.TrimSpace(string(out)))
+	if err != nil {
+		return fmt.Sprintf("bad TSA pid: %s", string(out))
+	}
+
+	// the service wasn't running. possible if the user didn't request TSA in the first place.
+	if outInt == 0 {
+		return ""
```

Comment:
This means that this gets re-evaluated every time there's a version query and the service wasn't present. We should probably switch to `sync.Once{}` if we decide to actually go this route.